plugins/GSdx/res/glsl/tfx_fs.glsl

```diff
@@ -860,12 +860,28 @@ void ps_main()
         // C.rgb = (C.rgb - 256.0f);
     // }
 // #endif
+#if PS_SCANMSK_TRANSPARENT
+    alpha_blend /= 2.0f;
+#endif
```

Comment:
This is just a hack.

The reason MGS doesn't work if you implement scanmsk is nothing to do with the blending equation. It has to do with our PCRTC implementation. You've simply bypassed actually addressing the problem and ignored the functionality of scanmsk to instead render at half transparency. This isn't a fix.

Reply:
Naturally Transparent mode was only an option to escape the interlaced look given by scaled and unscaled, but you're saying there's more to it with PCRTC? I was under impression that illusion of transparency was given by the TV's of the era but there's also something else that PS2 does that could be fixed too?

Reply:
Okay, at least I can see where you're coming from then.

No, PCRTC has two displays and MGS sticks the framebuffer in both of them and offsets one of them before compositing them together. PCSX2 just removes this offsetting as a part of another hack that removes blur.

Reply:
I've tackled GSRenderer::Merge a bit so that the offsetting would depend on a mode. Scaled looks pretty neat now with no noticeable blur. Please check if it's doable.